.github/workflows/package-client.yml

```diff
@@ -35,12 +41,15 @@ permissions:
 
 jobs:
   version:
+    if: ${{ inputs.version_patch_run_id == '' }}
     uses: ./.github/workflows/_parse_version.yml
     with:
       version: ${{ inputs.version }}
 
   webapp:
     needs: version
+    # Always run the job even if `version` job was skipped.
+    if: ${{ inputs.version_patch_run_id != '' && always() || success() }}
```

Comment:
Isn't the first condition redundant? Why not leaving just `always()` to match the comment?